src/main/java/com/transportcompany/transportcompany/models/entities/Order.java

```diff
@@ -1,6 +1,89 @@
 package com.transportcompany.transportcompany.models.entities;
 
+import jakarta.persistence.*;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+
+
+@Entity
+@Table(name = "orders")
 public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "order_id")
+    private String orderId;
+
+    @Column(name = "order_type")
+    private String orderType;
+
+    @Column
+    private String date;
+
+    @Column
+    private double price;
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false, cascade = CascadeType.MERGE)
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    public Order(){}
+    public Order(String orderId, String orderType, String date, double price, User user){
+        this.orderId = orderId;
+        this.orderType = orderType;
+        this.date = date;
+        this.price = price;
+        this.user = user;
+    }
+
+    public String getDate() {
```

Comment:
We can use Lombok to create getters and setters. Please use `@Data` annotation that will help you with that.